zero/meta-csp/recipes-cspd/csp-handler/files/yuyv_to_rgb.h

```diff
@@ -0,0 +1,98 @@
+#ifndef YUYV_TO_RGB_H
+#define YUYV_TO_RGB_H
+
+#ifdef __ARM_NEON__
+/*
+ * Based on libyuv/source/row_neon.cc [https://code.google.com/p/libyuv]
```

Comment:
1. Have you checked the license of libyuv?
2. Why did you copy & past a file from libyuv instead of lintegrate libyuv?